apps/heft/src/plugins/CopyStaticAssetsPlugin.ts

```diff
@@ -103,12 +103,18 @@ export class CopyStaticAssetsPlugin implements IHeftPlugin {
       heftConfiguration.rigConfig
     );
 
+    const destinationFolderNames: string[] = ['lib'];
+    // eslint-disable-next-line no-unused-expressions
+    typescriptConfiguration?.additionalModuleKindsToEmit?.forEach((emitModule) =>
+      destinationFolderNames.push(emitModule.outFolderName)
+    );
```

Comment:
This will get rid of the lint issue:

```suggestion
    for (const emitModule of typescriptConfiguration?.additionalModuleKindsToEmit || []) {
      destinationFolderNames.push(emitModule.outFolderName)
    }
```

Reply:
Interesting. I can do that, and will update with the changes. However, is this a bug in eslint? According to documentation I looked up yesterday on TypeScript optionals, the compiled output checks for undefined and creates a void statement if the value preceding `forEach` is undefined.

Reply:
We should probably enable an ESLint rule to deprecate `forEach`

Reply:
Deprecate forEach? Is that language feature going away or undesirable?

Reply:
I think @octogonz is referring to the `.forEach` iterator function. It's less efficient because each iteration is a function call with a new stack frame.